pkg/cmd/balances.go

```diff
@@ -33,19 +33,20 @@ var balancesCmd = &cobra.Command{
 			return errors.New("--config option is required")
 		}
 
+		// if config file exists, use the config loaded from the config file.
+		// otherwise, use a empty config object
+		var userConfig *bbgo.Config
 		if _, err := os.Stat(configFile); os.IsNotExist(err) {
-			return err
-		}
-
-		userConfig, err := bbgo.Load(configFile, false)
-		if err != nil {
-			return err
+			log.Info("config file doesn't exist, use the default config object")
+			userConfig = &bbgo.Config{}
+		} else {
```

Comment:
`err` here could be other error, inexist is just one kind of error, we should exclude this kind of error.

```
if err == nil { // load succeeded
} else if IsNotExists(err) { // does not exist
} else { // other errors
}
```

Reply:
You are right, I should take other situation into consideration.